Mathlib/Data/Rat/Cast/Lemmas.lean

```diff
@@ -3,8 +3,10 @@ Copyright (c) 2019 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
 -/
-import Mathlib.Data.Rat.Cast.Defs
 import Mathlib.Algebra.Field.Basic
+import Mathlib.Data.NNRat.Lemmas
```

Comment:
This seems like a big change in the import structure.

Reply:
I know why this is necessary and I will say two things:
1. It's avoidable in a way that doesn't involve unnecessarily splitting files
2. I am working on this way, but it's definitely a later PR and I don't really want to get into the weeds of that now